### ISSUE TITLE
CMake: proper way to include FindCapnProto.cmake file?

### ISSUE THREAD

**Issue by seeekr** [sandstorm-io/capnproto#380]
The way I'm using capnproto CMake support is by copying the cmake file included in the capnproto source into my project and including it manually. Is there a better / standard way to do this? Feels hackish. Snippet:

``` cmake
set(CapnProto_DIR "${CMAKE_CURRENT_SOURCE_DIR}/etc/cmake")
# follow regular instructions...
```

(EDIT: Also, having to rename the file to `CapnProtoConfig.cmake` when doing it this way.)

Is there a better way for including the official CMake file? It seemed to not get distributed with the header and library files when installing via homebrew.

ps. Awesome work with this library! Inspiring piece of software, great docs!

**Comment by pqu** [comment]
The changes that install the CMake find module haven't landed in a release yet, so copying the file is still necessary for the time being.

**Comment by seeekr** [comment]
Thanks for clarifying! I'm assuming my question will be answered then whenever that happens.

Feel free to do with this issue as you please then, i.e. keeping it open as a reminder for documenting the CMake integration, or just closing it if that's unnecessary.

Cheers

**Comment by kentonv** [comment]
Closing since this is "already fixed" in master, which will be released soon.

**Comment by seeekr** [comment]
awesome, thanks!

**Comment by StephanOpfer** [comment]
I just tried to install capnproto via github source code and I couldn't find it to copy the cmake-files into my system. Do you have a pointer to the instructions, that should do this? 

I followed these instructions to install: 
`git clone https://github.com/sandstorm-io/capnproto.git`
`cd capnproto/c++`
`autoreconf -i`
`./configure`
`make -j6 check`
`sudo make install`

And checked afterwards with:
`sudo updatedb`
`locate CapnProto`

**Comment by harrishancock** [comment]
@StephanOpfer this is only implemented in the CMake build -- see issue #523. I'm working on implementing it for the autotools build, too, and hope to have that finished soon.
